controller/src/main/java/io/pravega/controller/server/rpc/grpc/v1/ControllerServiceImpl.java

```diff
@@ -144,6 +146,67 @@ public void getCurrentSegmentsKeyValueTable(KeyValueTableInfo request, StreamObs
                 responseObserver);
     }
 
+    @Override
+    public void listKeyValueTablesInScope(Controller.KVTablesInScopeRequest request, StreamObserver<Controller.KVTablesInScopeResponse> responseObserver) {
+        String scopeName = request.getScope().getScope();
+        RequestTag requestTag = requestTracker.initializeAndTrackRequestTag(requestIdGenerator.get(),
+                "listKeyValueTables", scopeName);
+        log.info(requestTag.getRequestId(), "listKeyValueTables called for scope {}.", scopeName);
+
+        final AuthContext ctx;
```

Comment:
This can be simplified in a one-liner: `final AuthContext ctx = (this.grpcAuthHelper.isAuthEnabled()) ? AuthContext.current() : null;`

Reply:
done